### PR TITLE
update home value and zip4 to mask keys, add passing tests

### PR DESCRIPTION
**Pull request by ros3bud**
Solves #26

### CODE REVIEW

**Comment by ros3bud** [comment]
@alexkwolfe, LGTY? 

I removed lodash because it's only use was to merge vars into the default vars, but vars is getting set in the actual request.

File: spec/zip4_credit_score_spec.coffee [hunk 1]
```diff
@@ -62,9 +65,16 @@ describe 'Zip+4 Credit Score', ->
...
+    it 'should mask api keys on second invocation', ->
+      request2 = () ->   
+        outbound.request(@zipDefaultVars)
+      xml2 = parseXml(request2().body)  
+      assert.equal xml2.CUSTOMER.PARTNER_ID, '***********'
+      assert.equal xml2.CUSTOMER.CLIENT_ID, '*******************'
+
+
+    request = () ->
+      @zipDefaultVars =
```
Comment (alexkwolfe) [inline spec/zip4_credit_score_spec.coffee]: This is sort of a strange way to reuse the variables. You might consider factoring the variables out into a separate function call and then using that in both the `request()` function and in the test.

Also, when you are declaring a function that takes no parameters the convention is to leave out the parentheses.

**Comment by alexkwolfe** [comment]
Just that one comment. Otherwise LGTM.

File: spec/zip4_credit_score_spec.coffee (hunk 1 above)
Comment (ros3bud) [inline spec/zip4_credit_score_spec.coffee]: Thanks for catching the ()s. Will look into your suggestion.

**Pull request merged**